src/Framework/TestCase.php

```diff
@@ -1407,6 +1389,28 @@ public function addWarning(string $warning): void
         $this->warnings[] = $warning;
     }
 
+    /**
+     * Returns the normalized test name as class::method
+     *
+     * @return string[]
+     */
+    public function provides(): array
+    {
+        return $this->providedTests;
+    }
+
+    /**
+     * Returns a list of normalized dependency names, class::method
```

Comment:
```suggestion
     * Returns a list of normalized dependency names, class::method.
     *
```

Reply:
Added period and empty line after sentence

Reply:
There's a fixer for this, we could enable it. What do you think?

Reply:
I don't mind either way, really. What does this fixer do, try to guess where sentences end or just add periods and newlines?